cmd/bee/cmd/cmd.go

```diff
@@ -82,6 +83,12 @@ const (
 	optionReserveCapacityDoubling          = "reserve-capacity-doubling"
 	optionSkipPostageSnapshot              = "skip-postage-snapshot"
 	optionNameMinimumGasTipCap             = "minimum-gas-tip-cap"
+	optionAutoTLSEnabled                   = "autotls-enabled"
+	optionAutoTLSPort                      = "autotls-port"
```

Comment:
Alongside the port for WSS, we need the "nat-wss-addr" as we need to configure statically the nat for the WSS port as well. Also that requires the handling of its own resolver instance in libp2p.go for wss underlay addresses.